app/views/layouts/application.html.haml

```diff
@@ -8,5 +8,8 @@
     = csp_meta_tag
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_include_tag 'application'
+
+    = govuk_header(service_name: "LAA Fee Calculator User Interface")
+
```

Comment:
This doesn't need to go in with this PR but to fix the GDS layout you need to add stying to the body element, below. For example;

```haml
  %body.govuk-template__body
    .govuk-width-container
      = yield
```

You could also consider the 'common layouts' here: https://design-system.service.gov.uk/styles/layout/